src/main/routes/sessions.ts

```diff
@@ -1,24 +1,60 @@
-import * as express from 'express';
-import { v4 as uuidv4 } from 'uuid';
-import { Session } from '../models/Session';
-import {Sessions} from '../models/Sessions';
+import * as express from "express";
+import { v4 as uuidv4 } from "uuid";
+import { Session } from "../models/session";
+import { UserInfo } from "../models/userInfo";
+import { redisClient as redis } from "../app";
+import { IdamClient } from "../security/idam-client";
+const router = express.Router();
 
-const sessions: Sessions = {};
+router.get("/icp/sessions/:caseId", async (req, res) => {
+  const idam = new IdamClient();
```

Comment:
this would create a new idam client for every request, better to take the creation of the client outside the middleware